client/my-sites/stats/activity-log/index.jsx

```diff
@@ -0,0 +1,253 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { groupBy, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+import StatsFirstView from '../stats-first-view';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import StatsNavigation from '../stats-navigation';
+import ActivityLogDate from '../activity-log-date';
+
+class ActivityLog extends Component {
+
+	componentDidMount() {
+		window.scrollTo( 0, 0 );
+	}
+
+	update_logs( log ) {
+		const { translate } = this.props;
+		switch ( log.type ) {
+			// Scans
+			case 'security_threat_found':
+				log.subTitle = translate( 'Security Threat Found!' );
+				log.icon = 'notice';
+				log.actionText = translate( 'Fix it' );
+			// Backups
+			case 'site_backed_up':
+				log.subTitle = translate( 'Site Backed up' );
+				log.icon = 'checkmark';
+				log.status = 'is-success';
+				log.actionText = 'Restore';
+				break;
+			// Backups
+			case 'site_backed_up_failed':
+				log.subTitle = translate( 'Site Backed up' );
+				log.icon = 'notice';
+				log.actionText = 'Fix';
+				log.status = 'is-error';
+				break;
+			// Plugins
+			case 'plugin_activated':
+				log.subTitle = translate( 'Plugin Activated' );
+				log.icon = 'plugins';
+				log.actionText = 'Deactiavte';
+				break;
+			case 'plugin_deactivated':
+				log.subTitle = translate( 'Plugin Deactivated' );
+				log.icon = 'plugins';
+				log.actionText = 'Actiavte';
+				break;
+			case 'plugin_needs_update':
+				log.subTitle = translate( 'Plugin Update Available' );
```

Comment:
Alternate string suggestion: 
* `translate( 'There is a plugin update available.', 'There are plugin updates available.' )`&mdash; translations: **22**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log)